### PR TITLE
Update and improve the local linking instructions

### PR DESCRIPTION
**Pull request by thisisdano**
@jaredcunha Do these changes square with your install experience? I double-checked on my side as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,24 +51,25 @@ Here are a few other utility commands you may find useful:
...
+1. Clone the latest version of the [`uswds` repo](https://github.com/uswds/uswds/tree/develop).
+1. Run `npm install` to install the dependencies required for the package in the `uswds` directory.
+1. Run `npm run build` to create the built version of USWDS in the `uswds` directory.
+1. Run `npm link` in the _root level_ of the `uswds` directory on your local machine.
+1. Run `npm link uswds` in the _root level_ of the `uswds-site` directory on your local machine.
+1. Run `npm start` in the `uswds` directory, and nake a note of the `Local URL` that Fractal is serving.
+1. In another terminal window, set the `FRACTAL_BASE_URL` env var to the running Fractal instance for `uswds`. In your terminal window in the `uswds-site` directory, enter `export FRACTAL_BASE_URL="http://127.0.0.1:3000"` (or the `Local URL` noted above).
+1. In the `uswds-site` window, run `npm run serve` in the `uswds-site` directory to start the Jekyll server.
 1. Run `npm run watch` in both project directories to have changes automatically built and compiled on changes to any asset files.
```
Comment (thisisdano) [inline README.md]: We only need to run `npm run watch` in `uswds-site` as `npm start` already runs a watch in the `uswds` directory

File: README.md [hunk 2]
```diff
@@ -51,24 +51,25 @@ Here are a few other utility commands you may find useful:
...
 ### Fractal components
 
 USWDS uses the [fractal design system builder](http://fractal.build/) to organize and document the components. This documentation site pulls the components from fractal to showcase them on the site. This is done with a custom `fractal_component` Jekyll tag, which takes the full name of the fractal component as a parameter.
 
 ### Deployment and previews
 
-This site is deployed on [Federalist](https://federalist.fr.cloud.gov/), which automatically builds the site whenever commits are pushed to `master`.
+This site is deployed on [Federalist](https://federalist.18f.gov/), which automatically builds the site whenever commits are pushed to `master`.
 
 Federalist also builds public previews for each branch pushed to GitHub. For instance, to see the latest build of the `develop` branch, visit:
```
Comment (thisisdano) [inline README.md]: This needs to be updated as well

File: README.md (hunk 1 above)
Comment (jaredcunha) [reply README.md]: Ran through these steps on fresh clones and everything worked. Still newish, but if you save work on `uswds`, you still need to save a file on `uswds-site` in order to trigger the build and see the updated component, even if you didn't make changes to `uswds-site`.

Comment (thisisdano) [reply README.md]: OK, updated to clarify this point

**Pull request merged**